packages/graphics/src/styles/FillStyle.ts

```diff
@@ -11,6 +11,7 @@ export class FillStyle
 {
     /**
      * The hex color value used when coloring the Graphics object.
+     * Word-order is BGR
```

Comment:
```suggestion
     * Word-order is BGR, hex template is 0xRRGGBB.
```

Reply:
its inner field, if its modified - user has to know what is BGR :)